src/libs/SearchUtils.ts

```diff
@@ -22,6 +22,7 @@ import * as TransactionUtils from './TransactionUtils';
 import * as UserUtils from './UserUtils';
 
 type KeysOfFilterKeysObject = keyof typeof CONST.SEARCH.SYNTAX_FILTER_KEYS;
+type KeysOfRootKeysObject = keyof typeof CONST.SEARCH.SYNTAX_ROOT_KEYS;
```

Comment:
Please rename both to more simple: `FilterKeys`, `RootKeys` - this "keysOfKeys...object" naming is redundant IMO